Python/crossfire/crossfire/fogocruzado_utils.py

```diff
@@ -61,14 +69,9 @@ def extract_data_api(link):
     :return: pandas.DataFrame
         Result from the request API in pandas DataFrame format
     """
-    warn(
-        ("Extracting data from Fogo Cruzado's API."
-         "..."),
-        Warning)
-    headers = {'Authorization': fogocruzado_key()}
-    fogocruzado_request = requests.get(
-        url=link,
-        headers=headers)
+    warn(("Extracting data from Fogo Cruzado's API." "..."), Warning)
```

Comment:
Eu não acho que esses parênteses sejam necessários, podemos limpar o código:

```suggestion
    warn("Extracting data from Fogo Cruzado's API...", Warning)
```